src/azul/service/manifest_service.py

```diff
@@ -79,10 +79,8 @@ def __init__(self, storage_service: StorageService):
     def get_manifest(self, format_: str, filters: Filters):
         generator = ManifestGenerator.for_format(format_, self, filters)
         object_key, presigned_url = self._get_cached_manifest(generator, format_, filters)
-        if presigned_url is None:
```

Comment:
The removal of this conditional is a bug. It makes no sense to look for a cached manifest only to then ignore it and generate a new one.